ansible-scylla-node/tasks/common.yml

```diff
@@ -299,6 +299,45 @@
       when: token_distributor is defined
   when: start_scylla_service is defined and start_scylla_service|bool
 
+- name: Check if authentication is enabled
+  lineinfile:
+    path: /etc/scylla/scylla.yaml
+    regexp: '^(authenticator: (.*PasswordAuthenticator.*|.*TransitionalAuthenticator.*))'
```

Comment:
This is not enough - `authorizer` value needs to be checked too.

Reply:
Fixed